airflow/utils/logging.py

```diff
@@ -71,7 +71,7 @@ def read(self, remote_log_location, return_error=False):
             try:
                 s3_key = self.hook.get_key(remote_log_location)
                 if s3_key:
-                    return s3_key.get_contents_as_string().decode()
+                    return s3_key.get_contents_as_string()
```

Comment:
The default is `encoding='utf-8'` -- @TheSAS perhaps you could provide a test case?